README.md

```diff
@@ -52,6 +52,7 @@ Key | Description | Note
 | `azureStorageAccount` | string | 'mystorageaccount' |
 | `containerName` | string | 'photos' |
 | `containerSecurity` | string (optional) | 'blob' or 'container', defaults to blob |
+| `fileName` | function (optional) | function that recieves a file reference and allows customation of the file name |
```

Comment:
thank you for your contribution. Could you just please fix 2 typos in this line? 
I believe "customation" should be "customization", and "recieves" should be "receives"